etip/trackers/tests.py

```diff
@@ -1,6 +1,9 @@
 from django.test import TestCase, Client
+from django.core.management import call_command
+from django.core.management.base import CommandError
 from .models import Tracker
-from io import BytesIO
+from io import BytesIO, StringIO
+from mock import patch
```

Comment:
Replace this by `from unittest.mock import patch` and you won't need to add mock (and its dependencies) to ETIP requirements.

Reply:
Done; removed 3 deps!